### DISCUSSION TITLE
Uncaught SyntaxError: The requested module '/node_modules/.vite/deps/@clientio_rappid.js?v=6ad68c9e' does not provide an export named 'dia'

### DISCUSSION THREAD

**Discussion by babyandy0111** [clientIO/joint#2092] [Q&A] [answered]
### Introduction

https://resources.jointjs.com/tutorial/react-ts
I follow this ref and step by step do it. 

it always not working and console log is "Uncaught SyntaxError: The requested module '/node_modules/.vite/deps/@clientio_rappid.js?v=6ad68c9e' does not provide an export named 'dia'"

how can i do ?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by jamesgeorgewilliams** [comment, marked as answer]
I think this may be a duplicate of #1961 

Can you try to adapt your imports as mentioned above? Thanks

**Reply by babyandy0111** [reply]
wow, good job. thanks your answer. I solved.
